File: JS - Fundamentals/04. Functions - Exercise/addAndSubtract.js
function solve(a, b, c) {

    let combined = sum(a,b);
    return subtract(combined, c);

    function sum(x,y){
        return x + y;
    }

    function subtract(x,y){
        return x - y;
    }
}

console.log(solve(
    23,
    6,
    10
));

File: Java Fundamentals/04. Methods - Exercise/src/AddAndSubtract.java
import java.util.Scanner;

public class AddAndSubtract {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int x = Integer.parseInt(scan.next());
        int y = Integer.parseInt(scan.next());
        int z = Integer.parseInt(scan.next());

        System.out.println(subtract(sum(x,y), z));
    }

    public static int sum(int x, int y){
        return x + y;
    }

    public static int subtract(int x, int z){
        return x - z;
    }
}
